ibm/service/kubernetes/data_source_ibm_container_vpc_worker_pool.go

```diff
@@ -64,6 +64,14 @@ func DataSourceIBMContainerVpcClusterWorkerPool() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"kms_instance_id": {
```

Comment:
Can we add same arguments in data_source_ibm_container_vpc_cluster.go for "worker_pools" argument

Reply:
the cluster api doesnt give back the crk and kms instance id :(